content/sensu-go/5.6/installation/auth.md

```diff
@@ -302,10 +327,31 @@ type         | String
 default      | `"tls"`
 example      | {{< highlight shell >}}"security": "tls"{{< /highlight >}}
 
+| trusted_ca_file | |
+-------------|------
+description  | Path to an alternative CA bundle file to be used instead of the system's default bundle. This CA bundle is used to verify the server's certificate.
```

Comment:
Is there a format requirement? Should we call out that this is for use with mutual TLS?

Reply:
The CA bundle needs to be in PEM format.
`trusted_ca_file` is not strictly for mutual TLS, so no need to explicitly mention anything!